### DISCUSSION TITLE
Add Node via discovery

### DISCUSSION THREAD

**Discussion by Andyjb8** [jeisenbath/ansible-collection-solarwinds-orion#24] [Ideas]
Starting a discussion following up on this Issue: [Question/Issue Adding Custom Poller #16](https://github.com/jeisenbath/ansible-collection-solarwinds-orion/issues/16)

While researching my [Credential Set Issue #3](https://github.com/jeisenbath/ansible-collection-solarwinds-orion/issues/3), I ran across this: [Thwack Post](https://thwack.solarwinds.com/products/the-orion-platform/f/orion-sdk/25327/using-orion-credential-set-for-snmpv3-when-adding-node-through-sdk) where it was recommend adding nodes via discovery.  I am wondering if this is something worth putting on the roadmap?

**Comment by jeisenbath** [comment]
There's an example in the orionsdk-python repo too, I know it's possible I just personally haven't had the impetus to work it out. The only thing in my current scope of devices I would absolutely require discovery for is Windows Volumes.

I'm not against it being added in, in some fashion. Just from working with the interfaces module (which uses discovery), it is tricky and takes some time. Soon (this month if all goes well), I will be getting the ball rolling adding the collection to galaxy, and if that brings more community engagement this could be the first thing on the roadmap after.

**Comment by kbranvh** [comment]
Any update on this? It would be great to have. Thanks.

**Reply by jeisenbath** [reply]
My current focus is getting the collection up to par with the guidelines for inclusion to community inclusion, and then publishing to galaxy. There's only a few things left to do for that, but I'm sure this will be a popular request when more eyes are on it so likely will be worked on for the minor release after publishing.

**Comment by jeisenbath** [comment]
I'm closing this discussion topic and moving it to an Issue with an enhancement tag, since eventually the guidelines for Ansible community are to favor forum.ansible.com topics over github discussions. That will also keep more in line with the Contributing guidelines of having an issue open for any major changes.
